Stylesheets/Components/cards.css

```diff
@@ -0,0 +1,165 @@
+body {
+    background-color: var(--st-background-grey);
+}
+
+.card {
+    width: 15rem;
+    height: clamp(20rem, 25.5rem, 27rem);
+    background-color: var(--st-white);
+    border-radius: 4px;
+}
+
+.card-overlay {
+    width: 15rem;
+    height: 21.5rem;
+    background-color: var(--st-white);
+    border-radius: 4px;
+}
+
+.card-text-only {
+    width: 17rem;
+    height: 16rem;
+    background-color: var(--st-white);
+    border-radius: 4px;
+    padding: 0.5rem;
+
+}
+
+.shadow {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.card-horizontal {
+    width: fit-content;
+    background-color: var(--st-white);
+    display: flex;
+}
+
+.image-container {
+    height: 10rem;
+    width: 100%;
+    position: relative;
+}
+
+.card-badge {
+    position: absolute;
+    background-color: var(--warning-color);
+    color: var(--st-white);
+    padding: 5px;
+    border-radius: 2px;
+    top: 3px;
+    left: 0;
+}
+
+.card-dismiss {
+    position: absolute;
+    padding: 5px;
+    color: var(--st-black);
+    top: 2px;
+    right: 0;
+}
+
+.image-container img {
+    max-width: 100%;
+    height: auto;
+}
+
+.content-container {
+    padding: 0.2rem 0.3rem;
+}
+
+.reduced-price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 1rem;
+}
+
+.reduced-price h3 {
+    color: var(--success-color);
+}
+
+.rating {
+    display: flex;
+    align-items: center;
```

Comment:
u can add a utility class for declarations that are repeating, like display: flex in this case

Reply:
done thanks for pointing it out